RadixWallet/Core/DesignSystem/LinearGradients.swift

```diff
@@ -31,6 +31,21 @@ extension Gradient {
 	]
 }
 
+extension LinearGradient.App {
+	public var brand1: LinearGradient {
```

Comment:
Ended up not being used on these designs but decided to keep it as we will probably need it somewhere else (it is part of the [design system](https://www.figma.com/file/6MaKVYek0m9GKkd1xujB8P/Babylon-Wallet?type=design&node-id=405%3A3070&mode=design&t=dn5fkwpAWwUic1bK-1).